lib/llvm-backend/src/code.rs

```diff
@@ -2659,7 +2716,11 @@ impl ModuleCodeGenerator<LLVMFunctionCodeGenerator, LLVMBackend, CodegenError>
         Ok(())
     }
 
-    unsafe fn from_cache(artifact: Artifact, _: Token) -> Result<ModuleInner, CacheError> {
+    unsafe fn from_cache(_artifact: Artifact, _: Token) -> Result<ModuleInner, CacheError> {
+        Err(CacheError::Unknown(
+            "caching is broken for LLVM backend".into(),
```

Comment:
Is there a specific case where this is broken?

Reply:
Absolute address is used to reference the breakpoint table. Therefore caching won't work. To fix this, we need to dynamically relocate the breakpoint table.

Reply:
Okay, I think we should return an error here then only when metering is enabled.